addons/stock_landed_costs/models/stock_landed_cost.py

```diff
@@ -323,47 +321,24 @@ def _create_account_move_line(self, move, credit_account_id, debit_account_id, q
             credit_line['debit'] = -diff
         AccountMoveLine.create(debit_line)
```

Comment:
This could be a lengthy process:

aml_model.with_context(check_move_validity=False)